completion_aggregator/api/v1/views.py

```diff
@@ -351,10 +355,16 @@ def get(self, request, course_key):
             enrollments = UserEnrollments().get_course_enrollments(course_key)
             requested_fields.add('username')
             recalculate_stale = False
+            is_stale = False
         else:
             if not UserEnrollments(self.user).is_enrolled(course_key):
                 # Return 404 if effective user does not have an active enrollment in the requested course
                 raise NotFound()
+            is_stale = StaleCompletion.objects.filter(
+                username=self.user.username,
+                course_key=course_key,
+                resolved=False,
```

Comment:
Noting that the multi-field index on this table has `created` before `resolved`, so we'll end up doing a full scan of stale completions of a user in a course. I have no idea how big those set of rows look like, but I imagine not big when filtered down that much already, so probably nothing to do extra here.

Reply:
Yeah, the resolved StaleCompletions get deleted every hour, so there's not much extra work that needs to be done.  I'll take another look at indexes when things calm down, but it won't be a bottleneck.